.circleci/config.yml

```diff
@@ -6,15 +6,53 @@ commands:
       - restore_cache:
           name: Restore Yarn Cache
           keys:
-            - yarn-{{ checksum "yarn.lock" }}
+            - yarn-v2-{{ arch }}-{{ checksum "yarn.lock" }}
       - run:
           name: Yarn Install
-          command: yarn install --frozen-lockfile
+          command: YARN_CACHE_FOLDER=~/.cache/yarn yarn install --frozen-lockfile
       - save_cache:
           name: Save Yarn Cache
-          key: yarn-{{ checksum "yarn.lock" }}
+          key: yarn-v2-{{ arch }}-{{ checksum "yarn.lock" }}
           paths:
             - ~/.cache/yarn
+  react-native-install:
+    steps:
+    - run:
+        name: Install command line tools
+        command: |
+          sudo npm install -g react-native-cli
```

Comment:
Same here, we can just use `yarn react-native` instead when we're inside the repository directory and dev dependencies are installed

Reply:
I think I attempted this and it resulted in an `EACESS` error on the CI. Let me try and dig up an old build in circle and see.